app/src/main/java/com/mobile/tandil/javabaseproject/activity/MainActivity.java

```diff
@@ -0,0 +1,26 @@
+package com.mobile.tandil.javabaseproject.activity;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.os.Bundle;
+
+import com.mobile.tandil.javabaseproject.R;
+import com.mobile.tandil.javabaseproject.databinding.ActivityMainBinding;
+import com.mobile.tandil.javabaseproject.mvp.contract.ParkingContract;
+import com.mobile.tandil.javabaseproject.mvp.model.ParkingModel;
+import com.mobile.tandil.javabaseproject.mvp.presenter.ParkingPresenter;
+import com.mobile.tandil.javabaseproject.mvp.view.ParkingView;
+
+public class MainActivity extends AppCompatActivity {
+
+    private ActivityMainBinding binding;
+    private ParkingContract.Presenter presenter;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
```

Comment:
if you will use ViewBinding, you should instantiate it before this line and here change the resource by "binding.root"

Reply:
done